workflows/differentialabundance.nf

```diff
@@ -401,6 +402,26 @@ workflow DIFFERENTIALABUNDANCE {
             .first()
 
     } else {
+
+        // TODO: MOVE THE MODULE TO THE CORRECT SPOT LATER!
+        if ( params.contrasts.endsWith(".yaml") || params.contrasts.endsWith(".yml") ) {
+
+            ch_contrasts_file
+                .flatMap{ meta, yml ->
+                    YMLProcessing.parseContrastsFromYML(yml)
+                }
+                .set{ ch_contrast_dream }
+
+            DREAM_DIFFERENTIAL (
```

Comment:
I see that you built it to only trigger with YAML files, since we are looking into replacing csv contrasts files, I think this is OK. Leaving this comment here in case anyone spots this.